packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -1371,16 +733,16 @@ VirtualListBaseNative.displayName = 'ui:VirtualListBaseNative';
  * Example:
  * ```
  * onScrollStart = ({scrollLeft, scrollTop, moreInfo}) => {
- *     const {firstVisibleIndex, lastVisibleIndex} = moreInfo;
- *     // do something with firstVisibleIndex and lastVisibleIndex
+ *	 const {firstVisibleIndex, lastVisibleIndex} = moreInfo;
```

Comment:
Is there any reason to change spaces to tabs here?

Reply:
Fixed it.